Xamarin.Essentials/Calendars/Calendars.android.cs

```diff
@@ -622,7 +624,8 @@ static RecurrenceRule GetRecurranceRuleForEvent(string rule)
             {
                 var ruleOccurenceMonthDays = rule.Substring(rule.IndexOf("BYMONTHDAY=", StringComparison.Ordinal) + 11);
                 ruleOccurenceMonthDays = ruleOccurenceMonthDays.Contains(";") ? ruleOccurenceMonthDays.Substring(0, ruleOccurenceMonthDays.IndexOf(";", StringComparison.Ordinal)) : ruleOccurenceMonthDays;
-                recurranceRule.DayOfTheMonth = (uint)Math.Abs(Convert.ToInt32(ruleOccurenceMonthDays.Split(',').FirstOrDefault()));
+                uint.TryParse(ruleOccurenceMonthDays.Split(',').FirstOrDefault(), out recurranceRule.DayOfTheMonth);
+                ruleOccurenceMonthDays = Math.Abs(ruleOccurenceMonthDays);
```

Comment:
why are we reading out both recurranceRule.DayOfTheMonth and ruleOccurenceMonthDays?

Reply:
Have altered in latest PR, it shouldn't require a Math.Abs pass either since I'm TryParsing to a uint